addon/helpers/variation.js

```diff
@@ -1,9 +1,21 @@
 import Helper from '@ember/component/helper';
 import { inject as service } from '@ember/service';
+import { join } from '@ember/runloop';
+import { buildWaiter } from 'ember-test-waiters';
+
+let recomputeWaiter = buildWaiter('recompute-waiter');
 
 export default Helper.extend({
   launchDarkly: service(),
 
+  recompute() {
+    let token = recomputeWaiter.beginAsync();
+
+    this._super();
+
+    join(() => recomputeWaiter.endAsync(token));
```

Comment:
Using `join` because it will add to the end of the same queue that `recompute` itself uses above when `_super` is called, guaranteeing it will get called after the recompute is done.